src/main/java/com/kodilla/ecommercee/domain/User.java

```diff
@@ -0,0 +1,54 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@NoArgsConstructor
+@Setter
+@Getter
+@Entity
+@Table(name = "users")
+public class User {
+
+    public User(String username) {
+        this.username = username;
+    }
+
+    @Id
+    @NotNull
+    @GeneratedValue
+    @Column(name = "id", nullable = true)
+    public Long id;
+
+    @OneToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
```

Comment:
Wydaje mi się, że encja User powinna być też połączona z encją Order. Co prawda nie wynika to wprost z zaproponowanego diagramu, jednak zamówienie powinno być moim zdaniem przypisane do konkretnej osoby, a konkretna osoba powinna mieć możliwość wglądu w swoje zamówienie.